workshop-resources/keyboard-navigation/review-template.md

```diff
@@ -0,0 +1,209 @@
+# Collaborative keyboard navigation review
+
+Please add your findings as [GitHub suggestions](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request) so you are credited in the commit history!
+
+## Keyboard navigation accessibility review of [tool name]
+
+Reviewed via based on [this link we'll all be working off of]().
+
+- Review date:
+- Software version:
+- Keyboard language(s) used:
+- Any other details (ie. dependencies, github repo, etc.):
+
+### Authors
+
+Participants may add their name below if they would like to be listed publicly as a part of the review.
+
+1.
+2.
+3.
+4.
+5. 
+6.
+7.
+8.
+9.
+10. 
```

Comment:
This sign in area seems like a branch conflict nightmare waiting to happen. I would like to find a way for participants to voluntarily sign in and get clear credit for their work whether or not they get to writing a full review for commit, so this is the placeholder.

Reply:
This remains a problem in the latest commit.